test/05-supervisor-release.ts

```diff
@@ -51,4 +53,74 @@ describe('supervisor release', function() {
 			.delete(`/resin/supervisor_release(${supervisorReleaseId})`)
 			.expect(200);
 	});
+
+	describe('Devices running supervisor releases', () => {
+		let applicationId: number;
+		let device: fakeDevice.Device;
+		let fx: fixtures.Fixtures;
+		before(async () => {
+			fx = await fixtures.load('05-supervisor-release');
+			admin = fx.users.admin;
+			applicationId = fx.applications.app1.id;
+
+			device = await fakeDevice.provisionDevice(admin, applicationId);
+		});
+
+		it('should allow setting a device to a supervisor release', async () => {
+			await supertest(app, admin)
+				.patch(`/resin/device(${device.id})`)
+				.send({
+					should_be_managed_by__supervisor_release:
+						fx['supervisor-release']['5.0.1'].id,
```

Comment:
Could you do something like
```ts
let supervisorReleases: Dictionary<{ id: number }>
...
supervisorReleases = fx['supervisor-release']
```
please so we get some level of type enforcement (I would like to add types to the fixtures directly but that's far from straightforward, especially for `defaultFixtures` which are dynamically added)